code/modules/games/gamehelm.dm

```diff
@@ -0,0 +1,109 @@
+/obj/item/gamehelm
+	name = "\improper InUs game-helm"
+	desc = "The latest device in any self respecting gamer's arsenal, brought to you by the Ingi Usang Entertainment Corporation. Remember to hide it under your desk if the captain walks by."
+	desc_extended = "The Game-Helm was designed by a subsidiary of NanoTrasen, Ingi Usang Entertainment Co., with games \
+	being able to be purchased from an online marketplace hosted by InUs. Thousands of popular and obscure titles are available on the \
+	console. Besides being the perfect present, it's also capable of video streaming and sharing files over authorized \
+	connections. A quick and easy way to upload your latest montage to the extranet."
+	desc_info = "You can ALT-click the game-helm to open it up and turn it on. Click on the open device to play."
+	icon = 'icons/obj/gamehelm.dmi'
+	w_class = ITEMSIZE_SMALL
+	var/open = "open_white"
+	var/closed = "closed_white"
+
+/obj/item/gamehelm/Initialize()
+	. = ..()
+	icon_state = closed
```

Comment:
should probably `add_overlay()` the closed state button sprite here, otherwise they pop up when you first start the system.

Reply:
Good catch